src/include/storage/store/node_group.h

```diff
@@ -203,6 +205,7 @@ class NodeGroup {
     // `nextRowToAppend` is a cursor to allow us to pre-reserve a set of rows to append before
     // acutally appending data. This is an optimization to reduce lock-contention when appending in
     // parallel.
+    // TODO(Guodong): Remove this field.
```

Comment:
Are u going to resolve this TODO?

Reply:
Yeah. but I don't want to do it in this PR.